sources/models/src/lib.rs

```diff
@@ -340,6 +340,12 @@ struct BootSettings {
 #[model]
 struct AwsSettings {
     region: SingleLineString,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    config: Option<ValidBase64>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    credentials: Option<ValidBase64>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    profile: Option<SingleLineString>,
```

Comment:
The `skip_serializing_if` attribute and `Option` wrapping for the fields should be taken care of by `#[model]`. We only need to add them if we're adding any additional attribute that `#[model]` doesn't cover. See

https://github.com/bottlerocket-os/bottlerocket/blob/b3c35caa5808a15325c961e63bcc39bf163d8a3a/sources/models/model-derive/src/lib.rs#L143-L168

Reply:
Great! I just copied what I picked up somewhere else, so we may have a chance for some other clean up after this. Removed from this instance.